syntax = "proto3" ;
message Path {
string value = 1 ;
}

message PathRead {
  string value = 1;
  int32 offset = 2;
  int32 number_bytes = 3;
}

message PathFiles {
repeated string values = 2 ;
}
message FileData {
bytes data = 1;
}
service FS {
rpc ListFiles(Path) returns (PathFiles){};

rpc Read(PathRead) returns (FileData) {};
}

